Woof/WoofTests/ViewModel/SitterProfileViewModelTests/SitterProfileViewModelSaveTests.swift

```diff
@@ -127,4 +129,48 @@ final class SitterProfileViewModelSaveTests: XCTestCase {
         // Then
         XCTAssertEqual(requestCount, 1)
     }
+
+    func testFirstSaveChangesSitterIsSet() async {
+        // Given
+        MockURLProtocol.requestHandler = { request in
+            let response = try XCTUnwrap(
+                HTTPURLResponse(
+                    url: XCTUnwrap(request.url),
+                    statusCode: 200,
+                    httpVersion: nil,
+                    headerFields: nil
+                )
+            )
+            return (response, nil)
+        }
+
+        // When
+        let sitterIsSetBeforeSave = viewModel.sitterIsSet
+        await viewModel.save()
+
+        // Then
+        XCTAssertNotEqual(sitterIsSetBeforeSave, viewModel.sitterIsSet)
+    }
+
+    func testSitterIsSetIsNotChangedWhenSaveCalledTwice() async {
```

Comment:
You don't compare the initial value and the result value of the `sitterIsSet`, thus we can't say that "`sitterIsSet` is not changed". 
```suggestion
    func testSitterIsSetIsTrueWhenSaveIsCalledRepeatedly() async {
```

Reply:
Changed with 34c6879